soundbay/conf/checkpoint/defaults.yaml

```diff
@@ -1,3 +1,4 @@
 # @package _global_
 checkpoint:
-  path: null
\ No newline at end of file
+  path: null
+  resume: 'allow'
```

Comment:
Do we want this as a default flag?

Reply:
As far as I understood this will resume the previous run as long as the previous run id is explicitly provided. Think this is the behavior we want as it's explicit enough. 
https://docs.wandb.ai/guides/track/advanced/resuming